client.go

```diff
@@ -153,6 +153,37 @@ func (c *Client) DoMore(method string, in interface{}) (*ClientCall, error) {
 	return c.do(&req)
 }
 
+// OneWay is similar to Do, but does not expect any response from the service.
+func (c *Client) OneWay(method string, in interface{}) error {
+	req := clientRequest{
+		Method:     method,
+		Parameters: in,
+		Oneway:     true,
+	}
+
+	if req.Parameters == nil {
+		req.Parameters = struct{}{}
+	}
+
+	// This code does not use writeRequest: we do not want to create a new channel
+	// and wait for a response, because none will come back.
```

Comment:
Could we pass a nil channel to `writeRequest` instead, and only append to `c.pending` when `req.Oneway` is true?